apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -101,28 +130,28 @@ static void ble_operation_notification_cb(ble_client_operation_handle *handle, b
 
 static void ble_server_connected_cb(ble_conn_handle con_handle, ble_server_connection_type_e conn_type, uint8_t mac[BLE_BD_ADDR_MAX_LEN])
 {
-	RMC_LOG(RMC_TAG, "'%s' is called\n", __FUNCTION__);
-	printf("conn : %d / conn_type : %d\n", con_handle, conn_type);
-	printf("conn mac : %02x:%02x:%02x:%02x:%02x:%02x\n", mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
+	RMC_LOG(RMC_SERVER_TAG, "'%s' is called[%d]\n", __FUNCTION__);
```

Comment:
You used `%s` and `%d`, but passed only `__FUNCTION__`.
In this case, trash value can be printed.
Please check the parameters.

Reply:
You are right.
I fixed it